yaml2sbml/yaml_validation.py

```diff
@@ -26,6 +26,7 @@ def validate_yaml(yaml_file: str):
         yaml_in = yaml.full_load(yaml_contents)
 
     _validate_yaml_from_dict(yaml_in)
+    print('YAML file is valid ✅')
```

Comment:
Is there an option to suppress any messages? e.g. in PEtab, the linter returns nothing if validation was successful, unless the `-v` verbose flag is passed. This means if another program wants to determine whether a PEtab is valid, it can interpret the lack of a return message as success. Similarly, if a message by default is preferred in `yaml2sbml`, then an option to suppress success-related messages could provide the same convenience.

Reply:
But this would also mean that all messages like "Path to yaml..." and "Validating..." would need to be suppressed. In other programs, such messages are not displayed unless a verbose flag is passed.

Reply:
This is true, would be nice to have something like this :) I will dress this in another PR :)